docs-source/content/userguide/managing-domains/model-in-image/new-dynamicupdate-condition.md

```diff
@@ -0,0 +1,89 @@
+
+Server re-start is REQUIRED for the set of changes in progress.
+
+The following non-dynamic attribute(s) have been changed on MBeans
+that require server re-start:
+MBean Changed : com.bea:Name=oracle.jdbc.fanEnabled,Type=weblogic.j2ee.descriptor.wl.JDBCPropertyBean,Parent=[sample-domain1]/JDBCSystemResources[Bubba-DS],Path=JDBCResource[Bubba-DS]/JDBCDriverParams/Properties/Properties[oracle.jdbc.fanEnabled]
+Attributes changed : Value
+    Reason:                      BackoffLimitExceeded
+    Status:                      True
+    Type:                        Failed
+  Introspect Job Failure Count:  3
+  Message:                       Online update completed successfully, but the changes require restart and the domain resource  specified 'spec.configuration.model.onlineUpdate.onNonDynamicChanges=CancelUpdate'  option to cancel all changes if restart require. The changes are:
+
+Server re-start is REQUIRED for the set of changes in progress.
+
+The following non-dynamic attribute(s) have been changed on MBeans
+that require server re-start:
+MBean Changed : com.bea:Name=oracle.jdbc.fanEnabled,Type=weblogic.j2ee.descriptor.wl.JDBCPropertyBean,Parent=[sample-domain1]/JDBCSystemResources[Bubba-DS],Path=JDBCResource[Bubba-DS]/JDBCDriverParams/Properties/Properties[oracle.jdbc.fanEnabled]
+Attributes changed : Value
+  Reason:    BackoffLimitExceeded
+
+```
+
+If the changes involve non-dynamic mbean attributes, and you have specified 'CommitUpdateOnly' under `domain.spec.configuration.model.onlineUpdate.onNonDynamicChanges` or not set, you will see this               
+      
+```
+  Conditions:
+    Last Transition Time:  2021-01-20T15:09:15.209Z
+    Message:               Online update completed successfully, but the changes require restart and the domain resource specified 'spec.configuration.model.onlineUpdate.onNonDynamicChanges=CommitUpdateOnly' or not set. The changes are committed but the domain require manually restart to  make the changes effective. The changes are: Server re-start is REQUIRED for the set of changes in progress.
+
+The following non-dynamic attribute(s) have been changed on MBeans 
+that require server re-start:
+MBean Changed : com.bea:Name=oracle.jdbc.fanEnabled,Type=weblogic.j2ee.descriptor.wl.JDBCPropertyBean,Parent=[sample-domain1]/JDBCSystemResources[Bubba-DS],Path=JDBCResource[Bubba-DS]/JDBCDriverParams/Properties/Properties[oracle.jdbc.fanEnabled]
+Attributes changed : Value
+    Reason:                      Online update applied, introspectVersion updated to 82
+    Status:                      True
+    Type:                        ConfigChangesPendingRestart
+```
+and you will see the label in each pod  
+
+```Labels:       weblogic.configurationChangedPendingRestart=true
```

Comment:
maybe shorten the label a little so that it matches the condition name:  weblogic.configChangesPendingRestart
it might be best if you, Ryan, and I could meet to finalize the names